ui/packages/catalog-ui-search/src/main/webapp/js/OpenLayersGeometryUtils.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+/*jshint bitwise: false*/
+define([
+    'openlayers',
+    'properties',
+    './Common'
+], function (ol, properties, Common) {
+    return {
+        wrapCoordinatesFromGeometry: function(geometry) {
+            var type = geometry.getType();
+            var coordinates = [];
+            switch (type) {
+                case 'LineString':
+                    coordinates = geometry.getCoordinates();
```

Comment:
✏️ if you extract lines 36-38 into a function you can do something like:
geometry.setCoordinates(wrap(geometry.getCoordinates()));

then you can remove the second switch statement